### PR TITLE
Avatar component

### PR DESCRIPTION
**Pull request by Sundiata2**

### CODE REVIEW

File: src/scss/views/widgets/_avatar.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.avatar-wrapper {
+  display: inline-block;
+  vertical-align: middle;
```
Comment (alexander-daniel) [inline src/scss/views/widgets/_avatar.scss]: didn't wanna use `flex`?

File: viewer-app/root.tsx [hunk 2]
```diff
@@ -283,6 +284,7 @@ class Root extends React.Component<{}, {
 
         <ComponentSection title={'Images'}>
           <Image url="http://via.placeholder.com/350x150" style={testImageStyle} height={150} width={350}/>
+          <Avatar imgSrc="http://via.placeholder.com/150x150" height={40} width={40} circle={true} />
```
Comment (alexander-daniel) [inline viewer-app/root.tsx]: does it scale the 150x150 to fit inside the 40x40 or does it overflow?

File: src/scss/views/widgets/_avatar.scss (hunk 1 above)
Comment (Sundiata2) [reply src/scss/views/widgets/_avatar.scss]: I think it'll make the wrapper 100% width across by default if I do

File: viewer-app/root.tsx (hunk 2 above)
Comment (Sundiata2) [reply viewer-app/root.tsx]: scales to fit into 40X40

Comment (Sundiata2) [reply viewer-app/root.tsx]: height/width props take precedent over image url

File: src/scss/views/widgets/_avatar.scss (hunk 1 above)
Comment (alexander-daniel) [reply src/scss/views/widgets/_avatar.scss]: cool, just 🔪 the `vertical-align` rule if it's not doing nothin'!

**Comment by alexander-daniel** [comment]
🚚  it in!

**Pull request merged**